consul/config.sls

```diff
@@ -1,9 +1,7 @@
-{% from "consul/map.jinja" import consul with context %}
+{% from slspath+"/map.jinja" import consul with context %}
```

Comment:
Finally, I had a chance to get back on this.
So, I was wrong in my assumption that `tpldir` var could be a silver bullet. It doesn't work outside of SLS scope, imports, loads, etc and in this case it doesn't really different from `slspath`. I suspect it was mentioned in the doc by mistake and designed to be something internal... Maybe.

'nuff said I think these changes are good to go.